### ISSUE TITLE
web_responsive: search panel with groupby fails at low screensize

### ISSUE THREAD

**Issue by tdombos** [OCA/web#2016]
On v14 if the search panel has a field with groupby on, and the screensize is low, the action load fails with the following error message: 

```
TypeError: filter.groups.values.map is not a function or its return value is not iterable
    at PatchableClass.getActiveSummary (https://odoo14.hatter.hu/web/content/350-b1a1a09/web.assets_backend.js:6081:410)
    at QWeb.eval (eval at _compile (https://odoo14.hatter.hu/web/content/229-51c67ad/web.assets_common.js:969:178), <anonymous>:29:50)
    at QWeb.render (https://odoo14.hatter.hu/web/content/229-51c67ad/web.assets_common.js:958:20)
    at __owl__.renderFn (https://odoo14.hatter.hu/web/content/350-b1a1a09/web.assets_backend.js:1332:147)
    at Object.__owl__.renderFn (https://odoo14.hatter.hu/web/content/229-51c67ad/web.assets_common.js:1308:113)
    at PatchableClass.__render (https://odoo14.hatter.hu/web/content/229-51c67ad/web.assets_common.js:1288:33)
    at https://odoo14.hatter.hu/web/content/229-51c67ad/web.assets_common.js:1252:6
```

**Comment by Tardo** [comment]
Can you reproduce it on runbot? Please, enable debug mode with assets to see a more detailed error message.

**Comment by tdombos** [comment]
Confirmed on runbot. Detailed error message:

```
Traceback:
TypeError: filter.groups.values.map is not a function or its return value is not iterable
    at PatchableClass.getActiveSummary (http://3501554-14-0-3aab67.runbot2-3.odoo-community.org/web_responsive/static/src/js/web_responsive.js:602:59)
    at QWeb.eval (eval at _compile (http://3501554-14-0-3aab67.runbot2-3.odoo-community.org/web/static/lib/owl/owl.js:1766:28), <anonymous>:28:50)
    at QWeb.render (http://3501554-14-0-3aab67.runbot2-3.odoo-community.org/web/static/lib/owl/owl.js:1687:32)
    at __owl__.renderFn (http://3501554-14-0-3aab67.runbot2-3.odoo-community.org/web/static/src/js/model.js:482:20)
    at Object.__owl__.renderFn (http://3501554-14-0-3aab67.runbot2-3.odoo-community.org/web/static/lib/owl/owl.js:4743:20)
    at PatchableClass.__render (http://3501554-14-0-3aab67.runbot2-3.odoo-community.org/web/static/lib/owl/owl.js:4539:37)
    at http://3501554-14-0-3aab67.runbot2-3.odoo-community.org/web/static/lib/owl/owl.js:4283:26
```

**Comment by SplashS** [comment]
@tdombos Could you please describe steps to reproduce this error?
what did you mean here?
>>>  if the search panel has a field with groupby on

**Comment by tdombos** [comment]
Add the following definition to the contacts search view (res.partner.select)
```
<searchpanel>
    <field name="category_id" groupby="parent_id" select="multi" />
</searchpanel>
```
Resize the window making it less and less wide. At some point the error message above pops up.

**Comment by SplashS** [comment]
@tdombos mentioned PR fixes your case.
Could your please check this fix on your system or on PR runbot http://3510740-2060-34a1b5.runbot1.odoo-community.org/ ?

**Comment by tdombos** [comment]
Works good on runbot for me as well.
